packages/components/docs/sass.md

```diff
@@ -6369,15 +6372,45 @@ Include a type reset for a given body and mono font family
     -moz-osx-font-smoothing: grayscale;
   }
 
-  // IBM Plex uses semibold instead of bold, as a result we need to map
-  // tags that use `font-weight: bold` to the semibold value
-  strong {
-    font-weight: 600;
-  }
-
   code {
     font-family: $mono-font-family;
   }
+
+  h1 {
+    @include carbon--type-style('productive-heading-05');
+  }
+
+  h2 {
+    @include carbon--type-style('productive-heading-04');
+  }
+
+  h3 {
+    @include carbon--type-style('productive-heading-03');
+  }
+
+  h4 {
+    @include carbon--type-style('productive-heading-02');
+  }
+
+  h5 {
+    @include carbon--type-style('productive-heading-01');
+  }
+
+  p {
+    @include carbon--type-style('body-long-02');
+  }
+
+  a {
+    color: $link-01;
```

Comment:
@joshblack or @mattrosno what would be the proper way to include the token here? It seems like circle CI is failing bc of this. However, it seems to be getting the styles and everything.

Reply:
That token comes from the themes package. I don't think it's worth the trouble of having the type package depend on the themes package for a base anchor color. I'd hardcode the hex value or not set base anchor color at all.